tests/test_github.py

```diff
@@ -575,7 +577,14 @@ def test_fetch_more_pulls(self):
                                    'X-RateLimit-Remaining': '20',
                                    'X-RateLimit-Reset': '5'
                                })
-
+        httpretty.register_uri(httpretty.GET,
+                               GITHUB_REPO_URL + '/commits/53b970ee04bbc435842c14a2cbfdd623faf74a65/comments',
+                               body=[],
```

Comment:
Hey @valeriocos, I ain't sure if this is okay, since I am returning an empty list, please have a look.